src/ceph-detect-init/Makefile.am

```diff
@@ -55,11 +55,13 @@ EXTRA_DIST += \
 
 ceph-detect-init-all: ceph-detect-init/virtualenv
 
+CEPH_DETECT_INIT_VIRTUALENV := .
```

Comment:
I think best to have this hack on a per-module basis rather than a general thing shared among modules. Less chances for mistakes.